src/screens/flows/Send/utils.ts

```diff
@@ -1,18 +1,24 @@
 import { DEFAULT_FEE, XTC_FEE } from '@/constants/addresses';
 import { USD_PER_TC } from '@/utils/assets';
 
+// DELETE THIS CONST
```

Comment:
is this comment ok?

Reply:
😆 nice reminder to delete that. I'll delete it now